README.md

```diff
@@ -7,15 +7,15 @@ Helm umbrella chart (chart of charts) is an easy and really flexible way of inst
 | Umbrella Charts | Description | Status |
 |-----------------|-------------|--------|
 | [tyk-oss](./tyk-oss)                | Tyk Open Source | Stable              |
-| [tyk-single-dc](./tyk-single-dc)          | Tyk Self Managed (Single DC) | Beta            |
+| [tyk-stack](./tyk-stack)            | Tyk Self Managed (Single DC) | Beta            |
 | tyk-mdcb-control-plane | Tyk Self Managed (MDCB) Control Plane | Coming Soon     |
```

Comment:
What do you think renaming it to `control-plane` since it'll be named as control plane without `mdcb` reference in the naming
```suggestion
| tyk-control-plane | Tyk Self Managed (MDCB) Control Plane | Coming Soon     |
```

Reply:
Is it confirmed? If so can rename it

Reply:
yes, it is confirmed